keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -259,6 +341,15 @@ The Job controller doesn't add the environment variable if there is a name
 conflict with an existing environment variable. Users can specify other
 environment variables for the same annotation.
 
+<<[UNRESOLVED this deviates from the rest of the controllers ]>>
+The Pod name takes the form `$(job-name)-$(index)-$(random-string)`,
+which can be used for quickly identifying Pods for a specific index.
+<<[/UNRESOLVED]>>
+
+The Pod hostname takes the form `$(job-name)-$(index)` which can be used to
+address the Pod from others, when the Job is used in combination with a headless
+Service.
```

Comment:
Same comment as at the beginning - IIRC hostname is not about DNS entries, but rather about internal hostname within a pod.

Reply:
https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-hostname-and-subdomain-fields